src/preprocessing-service/preprocess.py

```diff
@@ -35,7 +35,7 @@ async def doc_generator(df):
     df["nulog_confidence"] = -1.0
     df["drain_matched_template_id"] = -1.0
     df["drain_matched_template_support"] = -1.0
-    df["anomaly_level"] = "info"
+    df["anomaly_level"] = "normal"
```

Comment:
`Normal` or `normal`?

Reply:
nulog inferencing is capitalized so will capitalize it here